doc/lightning-fundchannel.7.txt

```diff
@@ -31,6 +31,9 @@ than 16777215 satoshi.
 'urgent', 'normal' or 'slow' to use lightningd's internal estimates:
 'normal' is the default.
 
+'private' is an optional flag to keep this channel private, i.e. not
+broadcast a `channel_announcement` message.
```

Comment:
s/don't offer to create a `channel_announcement` message.  Note that we won't create a channel_announcement message unless the peer agrees to as well./?

Reply:
Damn, you're right!  Oops, I don't know how I got that idea then.  Thanks!